Sources/AppcuesKit/Data/Networking/NetworkClient.swift

```diff
@@ -173,11 +173,12 @@ internal class NetworkClient: Networking {
         _ urlRequest: URLRequest,
         completion: @escaping (_ result: Result<Void, Error>) -> Void
     ) {
-        let dataTask = config.urlSession.dataTask(with: urlRequest) { [weak self] _, response, error in
+        let dataTask = config.urlSession.dataTask(with: urlRequest) { [weak self] data, response, error in
             let url = (response?.url ?? urlRequest.url)?.absoluteString ?? "<unknown>"
             let statusCode = (response as? HTTPURLResponse)?.statusCode ?? -1
+            let logData = String(data: error?.data ?? data ?? Data.empty, encoding: .utf8) ?? ""
 
-            self?.config.logger.debug("RESPONSE: %{public}d %{public}@", statusCode, url)
+            self?.config.logger.debug("RESPONSE: %{public}d %{public}@\n%{private}@", statusCode, url, logData)
```

Comment:
@mmaatttt curious your thoughts on this part - in the function above this, we log the response/error body, but in this one we did not previously. I think it's helpful to have in either case here, granted this version does not actually deserialize the response into anything sent back to caller.

Reply:
Agreed. Probably just an oversight.